scripts/controller/projectsController.js

```diff
@@ -8,5 +8,23 @@
     $('#projects').fadeIn(500);
   };
 
+  projectsController.index = function(ctx, next) {
+    if(ctx.articles.length) {
```

Comment:
Here you refer to it as ctx.articles and you were thinking of setting that property in projectsController.showByType as ctx.projects.